packages/dashboard-frontend/src/store/Workspaces/devWorkspaces/actions/actionCreators/terminateWorkspace.ts

```diff
@@ -33,6 +35,16 @@ export const terminateWorkspace =
 
       const namespace = workspace.metadata.namespace;
       const name = workspace.metadata.name;
+      if (workspace.spec?.template?.attributes?.[DEVWORKSPACE_STORAGE_TYPE_ATTR] !== 'ephemeral') {
+        await DwApi.patchWorkspace(namespace, name, [
+          {
+            op: 'replace',
+            path:
+              '/spec/template/attributes/' + DEVWORKSPACE_STORAGE_TYPE_ATTR.replace(/\//g, '~1'),
+            value: 'ephemeral',
```

Comment:
@olexii4 I believe if we change storage before deletion PVCs will not be cleaned / deleted, right?

Reply:
@ibuziuk It depends. If I understand correctly, changing the storage type to `ephemeral` should affect storage as workspace termination. We make these changes to do it step by step.

Reply:
I have removed that part.

Reply:
I have fixed it